src/java.security.jgss/windows/classes/sun/security/krb5/internal/tools/Ktab.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2003, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2003, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2022

Reply:
Same goes for sun/security/krb5/internal/ktab/KeyTab.java.

Reply:
Yes.